src/overlay/ym2151_overlay.cpp

```diff
@@ -118,7 +118,7 @@ void draw_debugger_ym2151()
 					YM_debug_write(0x14, regs[0x14] | RES_MASK);
 				}
 				ImGui::TableNextColumn();
-				if (ImGui::SliderInt("Reload", &tim->reload, 0, TIM_MAX)) {
+				if (ImGui::SliderInt("Reload", &tim->reload, TIM_MAX, 0)) {
```

Comment:
Is this correct? The parameter order for SliderInt is `const char* label, int* v, int v_min, int v_max`, and TIM_MAX will always be greater than zero here.

Reply:
Oh, or, does this just flip the direction of the slider?